src/components/Forms/Form.js

```diff
@@ -43,39 +43,52 @@ class Form extends React.Component {
 
   componentWillReceiveProps(nextProps) {
     if (nextProps.serverErrors && !isEqual(this.state.serverErrors, nextProps.serverErrors)) {
-      this.setState({serverErrors: nextProps.serverErrors}, () => {
+      this.setState({ serverErrors: nextProps.serverErrors }, () => {
         this.setErrorsOnFormComponents(this.state.serverErrors)
       })
     }
   }
 
-  registerComponent = (component) => {
-    this.formComponents[component.props.name] = component
-  }
-
-  unregisterComponent = (component) => {
-    delete this.formComponents[component.props.name]
-    delete this.model[component.props.name]
-  }
-
   setErrorsOnFormComponents(serverErrors) {
-    Object.keys(serverErrors).forEach( (name) => {
+    Object.keys(serverErrors).forEach(name => {
       const component = this.formComponents[name]
       component.setState({
         isValid: false,
-        serverError: serverErrors[name]
+        serverError: serverErrors[name],
       })
     })
   }
 
+  handleSubmit = e => {
```

Comment:
manual change: `sort-comp` rule wants `set/get` then `on`/`handle`  after the lifecycle methods